docs/csharp/fundamentals/program-structure/snippets/top-level-statements-1/Program.cs

```diff
@@ -1,7 +1,16 @@
 ﻿using System.Text;
 
 StringBuilder builder = new();
-builder.AppendLine("Hello");
-builder.AppendLine("World!");
+builder.AppendLine("Hello,");
+builder.AppendLine("The following arguments are passed:");
 
 Console.WriteLine(builder.ToString());
+
+// Display the command line arguments using the args variable.
+foreach (var arg in args)
```

Comment:
Move line 7 beneath this `foreach` loop and then add the `args` to the `builder`.

Reply:
Good catch, thanks!